core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -378,7 +383,12 @@ trait AttributeComponent {
       }
 
       def clearAttributeScratchTableAction(transactionId: String) = {
-        sqlu"""delete from #${baseTableRow.tableName}_SCRATCH where transaction_id = $transactionId"""
+        val tableSuffix = if (baseTableRow.tableName == "ENTITY_ATTRIBUTE")
+          "TEMP"
+        else
+          "SCRATCH"
```

Comment:
it's only used twice, but might want to extract the `TEMP` vs `SCRATCH` determination into a shared function